pkg/extensions/management.go

```diff
@@ -0,0 +1,25 @@
+package extensions
+
+import (
+	jenkinsv1 "github.com/jenkins-x/jx/pkg/apis/jenkins.io/v1"
+	jenkinsclientv1 "github.com/jenkins-x/jx/pkg/client/clientset/versioned/typed/jenkins.io/v1"
+	"github.com/jenkins-x/jx/pkg/log"
+)
+
+func GetChildrenRecursively(ext jenkinsv1.Extension, configLookup map[string]jenkinsv1.ExtensionConfig, lookupByUUID map[string]jenkinsv1.Extension, extensionsClient jenkinsclientv1.ExtensionInterface, ns string, verbose bool) ([]jenkinsv1.Extension, error) {
```

Comment:
NIT: wrap lines when >120 chars

Reply:
Do you know how to make GoLand do this for me?

Reply:
![image](https://user-images.githubusercontent.com/1652511/48146791-b7764080-e2ad-11e8-9b64-e019bf415829.png)

Reply:
Very nice!